src/release.js

```diff
@@ -477,6 +505,9 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
          * @returns {Promise}
          */
         async promptToResolveConflicts() {
+            if (!interactive) {
+                return false;
```

Comment:
Since it's an `async` function, it's not needed. The returned value will be available in an asynchronous way, ie. `promptToResolveConflicts().then( value => {} )`